src/pages/to/[username].tsx

```diff
@@ -51,7 +56,7 @@ const SendTo = ({ username }: { username: string }) => {
             </div>
 
             {/* Message */}
-            <div className='flex min-h-[160px] flex-col justify-between space-y-5 px-5 py-10 sm:space-y-0 sm:px-10 sm:py-7'>
```

Comment:
why is height a fixed value here? couldnt we add spacing with margin or padding instead?

Reply:
Without the fixed height value, the height of the form updates when the message input unmounts. The fix is subtle but this ensures that there will be no height jumps. Video shows ui without the fixed height value.

https://user-images.githubusercontent.com/78056869/179350462-9f81358a-6fca-4c18-a7c2-8493e6b29ee3.mp4

Reply:
With fixed height value

https://user-images.githubusercontent.com/78056869/179350621-a23ff500-b9f5-4091-9979-3ce62fb51f2f.mp4

:

Reply:
what if the message expands to 2 lines?

Reply:
if that's the case, it's not noticeable compared to a one line message since the messages container will expand and will push the message input down

Reply:
@princejoogie wdyt? should we keep the fixed height or just rely on padding?

Reply:
if it expands when the message is 2 lines, this is okay to me

Reply:
My bad @princejoogie  I was talking about the form and not the message *facepalm* I have placed a fixed min height since the initial height was too small.

Without the min-h:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/78056869/179351486-1ef5e953-c0ef-48d6-b609-936da5a4f10d.png">

With the min-h:
<img width="631" alt="image" src="https://user-images.githubusercontent.com/78056869/179351500-18c4efa3-2dd8-4a8a-8961-3e0ac6d35036.png">

Reply:
oh its min height, my bad. thought it was a fixed height